cpp/generate_subscription_packets.sh

```diff
@@ -24,7 +24,10 @@ function Usage() {
 }
 
 
-generate_subscription_packets /usr/local/var/lib/tuefind/relbib_packets.conf /tmp/relbib_bundles.out
+readonly email_address=$1
```

Comment:
There is no check if $1 is set. Also Usage() is never called. And if Usage runs into exit 1, the trap will be called, and SendEmail will have an error => so Usage should be checked at least before the trap statement, maybe even before no_problems_found.